mountinfo/mounted_linux_test.go

```diff
@@ -207,11 +212,15 @@ func TestMountedBy(t *testing.T) {
 		}
 
 		checked = true
-		mounted, err = mountedByOpenat2(m)
-		if err != nil {
-			t.Errorf("mountedByOpenat2(%q) error: %v", m, err)
-		} else if mounted != exp {
-			t.Errorf("mountedByOpenat2(%q): expected %v, got %v", m, exp, mounted)
+
+		err = tryOpenat2(t)
```

Comment:
It doesn't matter much really, but calling it in a loop seems unclean -- e.g. it could make a reviewer think that something we do here might affect the value returned from `tryOpenat2`.

So I would move it out of the loop, doing something like
```go
openat2Supported = tryOpenat2(t) == nil
```
and below
```go
if openat2Supported {
   mounted, err = mountedByOpenat2(m)
   ...
```

Reply:
@thaJeztah PTAL ^^^